katalog/configs/infra/cluster-flow.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: ClusterFlow
+metadata:
+  name: infra
+spec:
+  filters:
+    - dedot:
+        de_dot_separator: "_"
+        de_dot_nested: true
+  match:
+    - exclude:
+        namespaces:
+          - ingress-nginx
+        labels:
+          app: ingress-nginx
+    - exclude:
+        namespaces:
+          - ingress-nginx
+        labels:
+          app: ingress
+          type: external
+    - exclude:
+        namespaces:
+          - ingress-nginx
+        labels:
+          app: ingress
+          type: internal
```

Comment:
just double-checking. We exclude the ingress namespace because it has its own index and formatting, correct?

Reply:
Yes, we have it's own output and flow